capabilities_matrix/_topics/cloud_providers.md

```diff
@@ -2,60 +2,60 @@
+| Snapshot Creation and Removal              | No                        | No              | No                          | Yes                        | Yes                                   | No                                |
+| Instance Compliance Enforcement            | Yes                       | Yes             | Yes                         | Yes                        | No                                    | No                                |
+| Instance Policy Enforcement                | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   | Yes                               |
+| Instance Power Operations                  | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   | Yes                               |
+| Instance Retirement                        | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   | Yes                               |
+| Alerts - Real Time                         | No                        | No              | No                          | Yes                        | No                                    | No                                |
+| Alerts - Instance Value Changed            | No                        | No              | No                          | No                         | No                                    | No                                |
+| Alerts - Instance Reconfigured             | No                        | No              | No                          | No                         | No                                    | No                                |
+| Integrate with Service Catalogs            | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   | Yes                               |
+| Instance Reconfiguration                   | No                        | No              | No                          | Yes                        | Yes                                   | No                                |
+| Volume Inventory                           | No                        | No              | Yes                         | Yes                        | Yes                                   | Yes                               |
+| Volume Creation/Deletion                   | No                        | No              | No                          | Yes                        | Yes                                   | Yes                               |
+| Instance Migration                         | No                        | No              | No                          | Yes                        | Yes                                   | No                                |
```

Comment:
Should we say N/A as this isn't supported by PowerVS - no somewhat gives the implication that it's a gap?

Reply:
(Migration)

Reply:
done